okio-testing-support/src/jsMain/kotlin/okio/testingSupportJs.kt

```diff
@@ -13,12 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-/**
- * See `fs.kt` for information on what this file does and how to keep it up-to-date.
- */
-@file:JsModule("os")
-@file:JsNonModule
 package okio
 
-internal external fun tmpdir(): String
+actual fun isBrowser(): Boolean {
```

Comment:
FYI, this is new! Added to ignore out tests that fail in browsers but not NodeJS. (Also want to dig into that, but not as urgently. Fails look like timeouts and not bugs.)